### ISSUE TITLE
No Info.plist file in application bundle or no NSPrincipalClass in the Info.plist file, exiting

### ISSUE THREAD

**Issue by LiquidLemon** [oNaiPs/dmenu-mac#26]
Hey, I just want to thank you for putting in the effort to make maintain this project, having a dmenu-like interface is the ideal for custom launcher scripts (I'm going to use it for a script to launch projects from my project directory in PyCharm). I'm having a problem running this program though. 

Here's all of the relevant info:
```
❯ dmenu-mac
2021-09-22 11:19:31.451 dmenu-mac[60055:1929984] No Info.plist file in application bundle or no NSPrincipalClass in the Info.plist file, exiting

❯ brew info dmenu-mac
dmenu-mac: 0.7.1
https://github.com/oNaiPs/dmenu-mac
/usr/local/Caskroom/dmenu-mac/0.7.1 (123B)
From: https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/dmenu-mac.rb
==> Name
dmenu-mac
==> Description
Keyboard-only application launcher
==> Artifacts
dmenu-mac.app (App)
/Applications/dmenu-mac.app/Contents/MacOS/dmenu-mac (Binary)
==> Analytics
install: 54 (30 days), 178 (90 days), 447 (365 days)

❯ sw_vers
ProductName:    Mac OS X
ProductVersion: 10.15.7
BuildVersion:   19H1323
```

**Comment by LiquidLemon** [comment]
Okay, disregard what I just wrote because launching it from the Applications folder actually works perfectly fine and so does using the path shown by `brew` (`/Applications/dmenu-mac.app/Contents/MacOS/dmenu-mac`). What's going on here? Is the `/usr/local/bin/dmenu-mac` some totally unrelated binary that I just have for whatever reason? Or is that a part of the project?

**Comment by LiquidLemon** [comment]
```
❯ md5sum /Applications/dmenu-mac.app/Contents/MacOS/dmenu-mac
de21bdd733002048d076e919b74e9a8d  /Applications/dmenu-mac.app/Contents/MacOS/dmenu-mac

❯ md5sum /usr/local/bin/dmenu-mac
de21bdd733002048d076e919b74e9a8d  /usr/local/bin/dmenu-mac
```

Okay, so they are the same thing but it just looks like the one in `/usr/local/bin` is missing some resources.
Sorry for rambling, I wasn't sure if it maybe was my fault it wasn't working 😅
I think it's as far as I can take it because I know next to nothing about native app development for MacOS.

**Comment by oNaiPs** [comment]
@LiquidLemon thanks for the report. This should be fixed in next 0.7.2, should be showing up in brew soon (https://github.com/Homebrew/homebrew-cask/pull/111695)

**Comment by LiquidLemon** [comment]
Works great, thanks!
